src/data-fetching/types.ts

```diff
@@ -5,6 +5,7 @@ export interface AlgoliaSettings {
 }
 
 export interface PrezlyNewsroomEnv {
+    IS_PREVIEW: string;
```

Comment:
Is it enough to just add it here? It should be as long as it's provided via deployment right?

Reply:
If you want to use this variable on the client side, you'll need to prefix it with `NEXT_PUBLIC_`.
Otherwise, should be good. But better to make it optional.

Reply:
I see. Should we then maybe add it next to https://github.com/prezly/theme-nextjs-bea/blob/main/@types/global.d.ts#L8 instead of here? 🤔